cypress/support/step_definition/course/editCourseSteps.spec.js

```diff
@@ -0,0 +1,89 @@
+import Courses from '../../pages/course/pageCourses'
+
+const courses = new Courses()
+
```

Comment:
here the comments are in between because it has the same flow as in the feature file, so to read the user user journey in step-definition in a same sequence as in the feature file, if we put all comments above and then the user flow would be changed as well, so we can adapt that in a follow up then once we have all features & step definition merged based on this structure.